package exception;

public class FinallyBlock1 {

public static void main(String[] args) {
	 
	
	try
	{

		String s=null;
		
		System.out.println(s.length());
		//System.out.println("try");
		
	}
		
	
	
//	catch (Exception e) {
//	
//		System.out.println(e);
//		}
//		
	finally{
		
		System.out.println("finally...........");
	}
	
	
	}

}
